multihost_test/bz_automation/script/pamtest.sh

```diff
@@ -0,0 +1,13 @@
+cat >/etc/pam.d/pamtest <<EOF
```

Comment:
Instead of providing a script to generate the PAM service file you should just provide the PAM service file and copy it to its location.

Reply:
Fixed